src/modules/ROOT/pages/changelog/202303/01.json

```diff
@@ -0,0 +1,889 @@
+        "This also improves consistency and UX."
+      ],
+      "how": [
+        "by changing the color to dark blue."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/8379518"
+      ]
+    },
+    {
+      "what": [
+        "the module dal settings Record ID field was prevented from being able to be disabled.",
+        "The unavailable badge added to system fields that are unchecked/unavailable from system field mapping and encoding is also changed.",
+        "In system field mapping and encoding, a 5 buttons grouped together are added to enable/disable system fields."
+      ],
+      "why": [
+        "To prevent users from disabling Record ID field on system field mapping and encoding.",
+        "Unavailable fields unchecked give users access to view system fields that are not available.",
+        "Also this was changed to give users easy access to enable/disable system fields based on the button options provided."
+      ],
+      "how": [
+        "By updating system field mapping logic to prevent Record ID from being disabled, this also applies to backend.",
+        "System view configurator with the unavailable badge was also updated.",
+        "Updated module dal settings to display button groups, with the logic to enable/disable fields that are related to a button"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/523/commits/07d41dcd8c1f40abf17f90868953eeb3603e0809" 
+      ]
+    },
+    {
+      "what": [
+        "the name of a page when was copied multiply times was changed to show the copy and after that the name of the page and a counter in brackets."
+      ],
+      "why": [
+        "to improve user experience.",
+        "If we copy a page 3-4 or more times, the naming gets confusing and too long."
+      ],
+      "how": [
+        "by filtering how many times a certain page with same name already exists and including 'copy of' prefix and number suffix.",
+        "Depending on the number of pages that match the same name, a counter is adjusted."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/644/commits/cbaca62b474fac5e9a19e9c500104fb93bf0ac9d" 
+      ]
+    },
+    {
+      "what": [
+        "workflow iterators to treat limit 0 parameter as no limit set."
+      ],
+      "why": [
+        "to make workflow iterators consistent with other searching/filtering interactions."
+      ],
+      "how": [
+        "by changing the limit in the code."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/cc24e622a" 
+      ]
+    },
+    {
+      "what": [
+        "the record list filter component updated with a unique id, which was previously the block id, also updated with a style of position: initial to prevent the record filter pop over z index issues."     
```

Comment:
I don't think I understand this

Reply:
I removed this since it is actually a fix for a new feature which we just introduced all together, so there is no need for a fix entry since this haven't been here before.